### PR TITLE
UBERF-4632: refactor activity classes structure

### PR DESCRIPTION
**Pull request by kristina-fefelova**
# Contribution checklist

## Checklist

* [ ] - UI test added to added/changed functionality?
* [ ] - Screenshot is added to PR if applicable ?
* [ ] - Does a local formatting is applied (rush format)
* [ ] - Does a local svele-check is applied (rush svelte check)
* [ ] - Does a local UI tests are executed [UI Testing](../tests/readme.md)
* [ ] - Does the code work? Check whether function and logic are correct.
* [ ] - Does Changelog.md is updated with changes?
* [ ] - Does the translations are up to date?
* [ ] - Does it well tested?
* [ ] - Tested for Chrome.
* [ ] - Tested for Safari.
* [ ] - Go through the changed code looking for typos, TODOs, commented LOCs, debugging pieces of code, etc.
* [ ] - Rebase your branch onto master and upstream branch
* [ ] - Is there any redundant or duplicate code?
* [ ] - Are required links are linked to PR?
* [ ] - Does new code is well documented ?

## Related issues

A list of closed updated issues

## Contributor requirements

* [ ] - Sign-off is provided. [DCO](https://github.com/apps/dco)
* [ ] - GPG Signature is provided. [GPG](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits)

### CODE REVIEW

**Comment by kristina-fefelova** [comment]
1.  `ChatMessage` moved  to `chunter`
2. `ActivityMessage`, `DocUpdateMessage`, `Reactions` moved to `activity`
3. `generateDocUpdateMessages` moved to `server-activity`

File: plugins/activity-resources/src/components/activity-message/ActivityMessageTemplate.svelte [hunk 1]
```diff
@@ -47,17 +50,17 @@
 
   const client = getClient()
   let allActionIds: string[] = []
-  const notifyActions = [
-    notification.action.MarkAsUnreadInboxNotification,
-    notification.action.MarkAsReadInboxNotification,
-    notification.action.DeleteInboxNotification
+  const notifyActions: Action[] = [
+    // notification.action.MarkAsUnreadInboxNotification,
```
Comment (haiodo) [inline plugins/activity-resources/src/components/activity-message/ActivityMessageTemplate.svelte]: Should we uncomment this?

**Review by haiodo — DISMISSED** [review]

File: models/chunter/src/index.ts [hunk 2]
```diff
@@ -177,6 +178,32 @@ export class TDirectMessageInput extends TClass implements DirectMessageInput {
...
+  @Index(IndexKind.FullText)
+    message!: string
+
+  @Prop(PropCollection(attachment.class.Attachment), attachment.string.Attachments, {
+    shortLabel: attachment.string.Files
+  })
+    attachments?: number
+
+  @Prop(TypeBoolean(), core.string.Boolean)
+  @Index(IndexKind.Indexed)
```
Comment (aonnikov) [inline models/chunter/src/index.ts]: nit: indexes on boolean attributes usually does not make sense because of low cardinality

**Review by aonnikov — DISMISSED** [review]

File: plugins/activity-resources/src/components/activity-message/ActivityMessageTemplate.svelte (hunk 1 above)
Comment (kristina-fefelova) [reply plugins/activity-resources/src/components/activity-message/ActivityMessageTemplate.svelte]: Thanks, I forgot about that
Removed excluding here and adjusted `visibilityTester`

File: models/chunter/src/index.ts (hunk 2 above)
Comment (kristina-fefelova) [reply models/chunter/src/index.ts]: Thanks, removed

**Review by haiodo — APPROVED** [review]

**Pull request merged**